frontend/models/feedback-task-model/dataset-field/DatasetField.model.js

```diff
@@ -11,6 +11,7 @@ class DatasetField extends Model {
       order: this.number(0),
       title: this.string(""),
       is_required: this.boolean(false),
+      use_markdown: this.boolean(true),
```

Comment:
We should have a generic `settings` attribute, instead of `use_markdown`. The `use_markdown` is only used for `TextField` components.